### DISCUSSION TITLE
Input type='text' in capitol letter in Notice Editor

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#447] [Q&A] [answered]
Hi lab !
Why input type = text are always in capital letter ?
![image](https://user-images.githubusercontent.com/52211037/236870189-5b6b0b75-a5a4-4e58-8574-1d66b6b409c2.png)

/* Input-fields of type "id" and "id-ref" are displayed with monospace uppercase type. */
.input-field.id > input[type='text'], .input-field.id-ref > input[type='text']  {
  font-family: monospace;
  text-transform: uppercase;
}

"id" : "BT-09(a)-Procedure",
   *********************
 },
  ****************,
    *****************,
    "pattern" : {
      "value" : "^CrossBorderLaw$",
      "severity" : "ERROR"
    }
  },

**Comment by rouschr** [comment, marked as answer]
Note that the entire project is meant as a demonstration for developers.
We provided a minimal design for **technical demonstration** purposes only, you can adapt or redo the entire project as you wish.
For this specific case it is to help see these id and id ref cases for technical reasons, so this for developers.
